examples/windows/windows-perfmon-proc-owner.yml

```diff
@@ -0,0 +1,10 @@
+integrations:
+  - name: nri-flex # We're telling the Infra agent to run Flex!
+    config: # Flex configuration starts here!
+      name: msPerfProcOwner
+      apis:
+        # Event Requests
+        - event_type: msPerfProcOwner # Name of the event in New Relic
+          commands:
+            - run: "C:/Program Files/New Relic/newrelic-infra/integrations.d/windows-perfmon-proc-owner.bat"
```

Comment:
please remove the batch file (`.bat`) dependency here and add the `shell` attribute to the config so you can execute the PowerShell script natively. Example [HERE](https://github.com/newrelic/nri-flex/blob/master/examples/windows/windows-ping-example.yml)

Reply:
Removed dependency on bat file